src/components/sections/people/ImportPane.scss

```diff
@@ -32,4 +32,27 @@
             }
          }
     }
+
+    .ImportPane-error {
+        .Button {
+            @include button;
```

Comment:
Perhaps we should use inline-button() mixin instead of button(), making it less demanding for action.